mysqlnd_azure.c

```diff
@@ -29,16 +29,23 @@
 #include "ext/mysqlnd/mysqlnd_statistics.h"
 #include "ext/mysqlnd/mysqlnd_connection.h"
 
+#include "utils.h"
+
 unsigned int mysqlnd_azure_plugin_id;
 struct st_mysqlnd_conn_data_methods org_conn_d_m;
 struct st_mysqlnd_conn_data_methods* conn_d_m;
 struct st_mysqlnd_conn_methods org_conn_m;
 struct st_mysqlnd_conn_methods* conn_m;
 
+FILE *logfile;
```

Comment:
set the value in explicit way. by default it's nullptr, but people don't know whether the default value is valid or not.